src/mixins/nft-collection.js

```diff
@@ -46,7 +47,7 @@ export default {
         description,
         priceLabel,
         price,
-        value: -1,
+        value: 0,
```

Comment:
What does this mean?

Reply:
Price Index, doesn't mean anything in collection but now having it smaller than 0 trigger a warning

Reply:
I see